Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/jvm/CommandLine.kt

```diff
@@ -22,6 +22,7 @@ import kotlinx.cli.*
 
 const val HEADER_FILTER_ADDITIONAL_SEARCH_PREFIX = "headerFilterAdditionalSearchPrefix"
 const val NODEFAULTLIBS = "nodefaultlibs"
+const val NOENDORSEDLIBS = "noendorsedlibs"
```

Comment:
In Kotlin/JVM the similar compiler options are called `-no-stdlib` and `-no-reflect`. Makes sense to keep consistency here and consider calling it `-no-endorsedlibs` or `-no-endorsed-libs`.

Reply:
I named in same way as we already have flag for platform and standard libs. If change style, then `-nodefaultlibs` and  `-nostdlib` also should be renamed. We should deprecated old forms and add new ones.
@olonho what do you think?

Reply:
Yes, let's make them consistent.

Reply:
But this option could be called -no-endorsed-libs already.